cyber/readme.md

```diff
@@ -4,3 +4,111 @@ _**A**_
    <dd>	A method that is used to confirm the identity of a user, through various means such as password entry or fingerprint scan. This allows them to have access to the system or files.  	</dd>
 </dl>
 
+_**D**_
```

Comment:
```suggestion
# D
```

Reply:
Please change all of the letters to headings. This way you can link to them from the top of the page.

Reply:
Done